version-select.js

```diff
@@ -0,0 +1,164 @@
+window.addEventListener("DOMContentLoaded", function() {
+    window.versionPages = {};
+    var VERSION = window.location.pathname.split("/")[1];
+    var VERSION_LATEST = '.';
+
+    function removePrefix(str, prefix) {
+        var hasPrefix = str.indexOf(prefix) === 0;
+        return hasPrefix ? str.substr(prefix.length) : str.toString();
+    }
+
+    function populateVersionSitemap(version) {
+        var versionPath = version === VERSION_LATEST ? "" : "/" + version;
+        window.versionPages[version] = [];
+
+        var xhrSitemap = new XMLHttpRequest;
+        var sitemapURL = window.location.origin + versionPath + '/sitemap.xml';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/codacy/docs/pullRequest?prid=26920)
```suggestion
865edb930f1cdc2e8417d577bb76a92144d19f68978c1438ff3e41ce609804588c3145b2625e1634bb594e3873af77c0766addcfc18ced02eba5f766bc59e6538712b7521f6ba71b956b86c9ebbcaa9b
```